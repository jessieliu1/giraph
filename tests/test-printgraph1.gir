int main() {
	{
		graph<bool> g = [A:true --  B:false];
		g.print();
	}

	{
		digraph<float> g = [A:1.0 -> B:1.5 <-> C:100.6];
		g.print();
	}

	{
		wegraph<int> g = [A:1 -{6}- B:2 -{100}- C:3];
		g.print();
	}

	{
		wedigraph<string> g = [A:"we" -{6}-> B:"out" <-{100}-> C:"here"];
		g.print();
	}
	return 0;
}