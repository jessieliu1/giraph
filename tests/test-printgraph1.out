vertex: true
adjacencies: false 
vertex: false
adjacencies: true 

vertex: 1.000000
adjacencies: 1.500000 
vertex: 1.500000
adjacencies: 100.599998 
vertex: 100.599998
adjacencies: 1.500000 

vertex: 1
adjacencies: (2, weight: 6) 
vertex: 2
adjacencies: (1, weight: 6) (3, weight: 100) 
vertex: 3
adjacencies: (2, weight: 100) 

vertex: we
adjacencies: (out, weight: 6) 
vertex: out
adjacencies: (here, weight: 100) 
vertex: here
adjacencies: (out, weight: 100) 

